org.eclipse.wb.rcp/plugin.xml

```diff
@@ -377,7 +377,9 @@
 	<!-- ======================================================== -->
 	<!-- NLS sources -->
 	<!-- ======================================================== -->
-	<extension point="org.eclipse.wb.core.nlsSources">
+	<extension point="org.eclipse.wb.core.nlsSources">
+		<binding toolkit="org.eclipse.wb.rcp"
+			description="org.eclipse.wb.core.nls.bundle.pure.activator.ActivatorSource"/>

```

Comment:
The description form seems weird. should that not be a human readable text?

Reply:
Yes, you are correct - a description "Activator plugin messages"

Reply:
I cannot rule out that this is used somewhere to lookup the classes of the extension points. As I don't fully understand this part of WB, I would rather not risk breaking something; and say in the style of all other WB implementation classes.